packages/wrangler/src/config/environment.ts

```diff
@@ -265,6 +265,13 @@ interface EnvironmentInheritable {
 	 * @inheritable
 	 */
 	logpush: boolean | undefined;
+
+	/**
+	 * Specify how the worker should be located to minimize round-trip time.
+	 *
+	 * More details: https://developers.cloudflare.com/workers/platform/smart-placement/
+	 */
+	placement: { mode?: "off" | "smart" } | undefined;
```

Comment:
Why is `mode` optional? Why would a user write `placement = {}`?

Reply:
I think this is a remnant of a previous iteration of this PR before it got scoped down, I'll make mode required.